src/gui/screen_printing.cpp

```diff
@@ -23,7 +24,8 @@
 #define BUTTON_PAUSE 1
 #define BUTTON_STOP  2
 
-#define POPUP_MSG_DUR_MS 5000
+#define POPUP_MSG_DUR_MS       5000
+#define MAX_END_TIMESTAMP_SIZE (MAX_DATE_STR_SIZE + MAX_TIME_STR_SIZE + 5) // "dd.mm.yyyy at hh:mm:ss" + safty measures for 3digit where 2 digits should be
```

Comment:
avoid macros
use static const size_t for size constants

Reply:
Why? I think code consistency is more important

Reply:
if you can, avoid macros in C++ code (code safety)